pkg/kubectl/describe.go

```diff
@@ -1395,6 +1396,32 @@ func DescribeEvents(el *api.EventList, w io.Writer) {
 	}
 }
 
+// DeploymentDescriber generates information about a deployment.
+type DeploymentDescriber struct {
+	client.Interface
+}
+
+func (dd *DeploymentDescriber) Describe(namespace, name string) (string, error) {
+	d, err := dd.Experimental().Deployments(namespace).Get(name)
```

Comment:
I would agree with @smarterclayton here. Keeping copies of other objects' state in the Deployment object's status will either be frequently out of date or will lead to lots of extraneous updates to apiserver. At the very most, perhaps the matching RC names should be in the status (and, during a deployment, both the old RC's and the new RC's).